### PR TITLE
grammar, typo, link fixes

### PR DESCRIPTION
**Pull request by enortonAtAdobe**
## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: what-to-implement/shopping-cart.md [hunk 1]
```diff
@@ -40,21 +40,21 @@ adbe('event', {
...
+* `productName` - The display name of the product.
 * `productImage` - A URL for the image. This is used when you want to display an image in the product. \(e.g. recommendations use cases\)
-* `productPrice` - The full list price of the product. Should be sent when a product is added to cart as well as when a product is purchased. 
-* `productCategory` - The category that was used to find the product \(if a product can be listed in multiple categories\). 
-* `productSubCategory` - The subcategory \(if any\) that was used to find a product. \(additional keys may be added to indicate further levels of sub categorization if you have them, remember to add them to your XDM\). 
+* `productPrice` - The full list price of the product. Should be sent when a product is added to cart as well as when a product is purchased.
+* `productCategory` - The category that was used to find the product \(if a product can be listed in multiple categories\).
+* `productSubCategory` - The subcategory \(if any\) that was used to find a product. \(additional keys may be added to indicate further levels of sub categorization if you have them, remember to add them to your XDM\).
```
Comment (Aaronius) [inline what-to-implement/shopping-cart.md]: * "sub categorization" => "subcategorization"
* The text in the parenthesis is grammatically incorrect. I'd probably do: "The subcategory \(if any\) that was used to find a product. Additional keys may be added to indicate further levels of subcategorization if you have them (remember to add them to your XDM schema\)."

File: what-to-implement/shopping-cart.md [hunk 2]
```diff
@@ -40,21 +40,21 @@ adbe('event', {
 
 ### General Product Attributes
 
-You will want to include the following information in your each of the events above. 
+You will want to include the following information in your each of the events above.
 
-* `productId` - The product ID, this is usually how you refer to the product internally. Customers will usually add product catalog data to this ID. 
-* `productName` - The display name of the product. 
+* `productId` - The product ID, this is usually how you refer to the product internally. Customers will usually add product catalog data to this ID.
```
[the remark this answers is not in the document]
Comment (Aaronius) [reply what-to-implement/shopping-cart.md]: "Customers will usually add product catalog data to this ID." <- I'm not really sure what this means...

File: what-to-implement/shopping-cart.md [hunk 3]
```diff
@@ -40,21 +40,21 @@ adbe('event', {
...
+* `productName` - The display name of the product.
 * `productImage` - A URL for the image. This is used when you want to display an image in the product. \(e.g. recommendations use cases\)
```
Comment (Aaronius) [inline what-to-implement/shopping-cart.md]: * "This is used when you want to display an image in the product. \(e.g. recommendations use cases\)" => "This is used when you want to display an image in the product \(e.g., recommendations use cases\).

File: what-to-implement/shopping-cart.md [hunk 4]
```diff
@@ -40,21 +40,21 @@ adbe('event', {
 
 ### General Product Attributes
 
-You will want to include the following information in your each of the events above. 
+You will want to include the following information in your each of the events above.
```
Comment (Aaronius) [inline what-to-implement/shopping-cart.md]: * "in your each of the events" => "in each of the events"

File: what-to-implement/shopping-cart.md [hunk 5]
```diff
@@ -5,29 +5,29 @@ description: How to add data if you have products or a shopping cart
...
+This documentation is for a library and a service that is in Alpha and should not be used for production use-cases.
 {% endhint %}
 
-If you have products on your site then this is a default set of things the you will want to send to enable the most capabilities from Adobe. Though this is a suggestion it will provide you a very strong set of data right from the start. 
+If you have products on your site then this is a default set of things the you will want to send to enable the most capabilities from Adobe. Though this is a suggestion it will provide you a very strong set of data right from the start.
 
 ## Actions Related to Products
 
-There are certain actions that make sense to capture in relation to the purchase cycle. Here is the list of those events. 
+There are certain actions that make sense to capture in relation to the purchase cycle. Here is the list of those events.
```
Comment (Aaronius) [inline what-to-implement/shopping-cart.md]: Maybe "Here is a list of events that pertain to a purchase cycle that you may wish to capture."

File: what-to-implement/shopping-cart.md [hunk 6]
```diff
@@ -131,17 +131,14 @@ adbe('event', {
...
 ### Subscriptions
 
-If your products include a subscription or are a subscription you also want to add these events and fields. 
+If your products include a subscription or are a subscription you also want to add these events and fields.
 
 #### Events
 
 * `subscribe` - This will be used instead of purchase
 * `updateSubscription` - Used when a customer moves between service tiers
 * `cancelSubscription` - When a customer cancels their subscription.
```
Comment (Aaronius) [inline what-to-implement/shopping-cart.md]: This starts with "When" but the previous bullet starts with "Used when". They should probably be consistent.

**Comment by enortonAtAdobe** [comment]
Thanks, @Aaronius!  Made updates from your suggestions.

**Comment by Aaronius** [comment]
👍 Thanks @enortonAtAdobe

**Pull request merged**